dashboard/test/test_helper.rb

```diff
@@ -156,46 +156,28 @@ def stub_project_source_data(channel_id, code: 'fake-code', version_id: 'fake-ve
   include ActiveSupport::Testing::TransactionalTestCase
   include ActiveSupport::Testing::SpecSyntax
   include CaptureQueries
+  include Curriculum::SharedCourseConstants
 
-  def seed_deprecated_unit_fixtures
-    # Some of the functionality we're testing here relies on Scripts with
-    # certain hardcoded names. In the old fixture-based model, this data was
-    # all provided; in the new factory-based model, we need to do a little
-    # prep.
-    #
-    # NOTE for any future developers: please DO NOT add new scripts to this
-    # list. This exists to provide backwards compatibility to old tests which
-    # are dependent on factory-provided content. If you are writing new tests,
-    # please make sure that they are instead relying on factory-provided
-    # content.
-    tested_script_names = [
-      'ECSPD',
-      'allthethings',
-      Unit::COURSE1_NAME,
-      Unit::COURSE4_NAME,
-      Unit::FLAPPY_NAME,
-      Unit::FROZEN_NAME,
-      Unit::HOC_NAME,
-      Unit::PLAYLAB_NAME,
-      Unit::TWENTY_HOUR_NAME
```

Comment:
Are we now able to remove some or all of these from `dashboard/test/fixtures/unit.yml` as a follow-up?

Reply:
That's the goal! I've listed that as future work.

Reply:
I did a quick search and while there are a few units that could require only minimal work to remove from the test fixtures, it doesn't look like there are any which are ready to remove as of this PR.